### PR TITLE
Added shell script used in Azure Pipelines build

### PR DESCRIPTION
**Pull request by yradsmikham**
#28

### CODE REVIEW

File: gitops/azure-devops/build.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+#git clone https://github.com/Microsoft/fabrikate
+#cd fabrikate/examples/getting-started
+fab generate prod
+echo "FAB GENERATE PROD COMPLETED"
+ls -a
+
+cd /home/vsts/work/1/s/
+echo "GIT CLONE"
+git clone https://github.com/yradsmikham/walmart-k8s.git
```
Comment (andrebriggs) [inline gitops/azure-devops/build.sh]: @yradsmikham Perhaps make the reference to repo a variable

File: gitops/azure-devops/build.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+cd walmart-k8s
```
Comment (andrebriggs) [inline gitops/azure-devops/build.sh]: This is hardcoded. Can we make it relative to whatever an operator is using?

File: gitops/azure-devops/build.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+echo "GIT CHECKOUT"
+git checkout master
+echo "GIT STATUS"
+git status
+echo "Copy yaml files to repo directory..."
+rm -rf prod/
+cp -r /home/vsts/work/1/s/generated/* .
+ls /home/vsts/work/1/s/walmart-k8s
```
Comment (samiyaakhtar) [inline gitops/azure-devops/build.sh]: This directory should maybe be renamed to something generic, that doesn't need a variable?

File: gitops/azure-devops/build.sh (hunk 2 above)
Comment (samiyaakhtar) [inline gitops/azure-devops/build.sh]: This needs to come from a variable

File: gitops/azure-devops/build.sh (hunk 3 above)
Comment (andrebriggs) [inline gitops/azure-devops/build.sh]: Hardcoded reference to "walmart-k8s"

File: gitops/azure-devops/build.sh [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+git config user.name "Automated Account"
+
+echo "GIT COMMIT"
+git commit -m "Updated k8s manifest files"
+echo "GIT STATUS" 
+git status
+echo "GIT PULL" 
+git pull
+echo "GIT PUSH"
+git push https://$ACCESS_TOKEN@github.com/yradsmikham/walmart-k8s.git
```
Comment (andrebriggs) [inline gitops/azure-devops/build.sh]: Hard coded reference to your repo

File: gitops/azure-devops/build.sh [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+git status
+echo "Copy yaml files to repo directory..."
+rm -rf prod/
+cp -r /home/vsts/work/1/s/generated/* .
+ls /home/vsts/work/1/s/walmart-k8s
+echo "GIT ADD"
+git add *
+
+#Set git identity 
+git config user.email "admin@azuredevops.com"
```
Comment (andrebriggs) [inline gitops/azure-devops/build.sh]: @yradsmikham We should probably allow a user to set this to whatever they want. Thoughts? cc @samiyaakhtar

File: gitops/azure-devops/build.sh [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+echo "Downloading Fabrikate..."
+echo "Latest Fabrikate Version: $LATEST_VERSION"
+#wget "https://github.com/Microsoft/fabrikate/releases/download/0.1.2/fab-v0.1.2-linux-amd64.zip"
+wget "https://github.com/Microsoft/fabrikate/releases/download/$LATEST_VERSION/fab-v$LATEST_VERSION-linux-amd64.zip"
+#unzip fab-v0.1.2-linux-amd64.zip -d fab
+unzip fab-v$LATEST_VERSION-linux-amd64.zip -d fab
+export PATH=$PATH:/home/vsts/work/1/s/fab
+fab install
+
+#git clone https://github.com/Microsoft/fabrikate
```
Comment (andrebriggs) [inline gitops/azure-devops/build.sh]: @yradsmikham remove commented out code

File: gitops/azure-devops/build.sh (hunk 5 above)
Comment (yradsmikham) [reply gitops/azure-devops/build.sh]: Sure, we make this change in the future.

**Pull request merged**